### PR TITLE
test: Add run linking tests

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: cypress/e2e/5-ndv.cy.ts [hunk 1]
```diff
@@ -163,5 +163,70 @@ describe('NDV', () => {
...
+	it('can link and unlink run selectors between input and output', () => {
+		cy.fixture('Test_workflow_5.json').then((data) => {
+			cy.get('body').paste(JSON.stringify(data));
+		});
+		workflowPage.actions.zoomToFit();
+		workflowPage.actions.executeWorkflow();
+		workflowPage.actions.openNode('Set3');
+
+		ndv.getters.inputRunSelector()
+			.should('exist')
```
Comment (MiloradFilipovic) [inline cypress/e2e/5-ndv.cy.ts:177]: Is this necessary? I think next `find` should fail if the element is not there.

**Review by MiloradFilipovic — DISMISSED** [review]
Looks good. Left a minor comment but good to go without it.

File: cypress/e2e/5-ndv.cy.ts (hunk 1 above)
Comment (mutdmour) [reply cypress/e2e/5-ndv.cy.ts:177]: if I understood correctly, this is like a wait until the element exists..

File: cypress/e2e/5-ndv.cy.ts [hunk 2]
```diff
@@ -163,5 +163,70 @@ describe('NDV', () => {
 			ndv.getters.outputPanel().find('[class*=_pagination]').should('not.exist');
 			ndv.getters.outputPanel().find('[data-test-id=run-data-schema-item] [data-test-id=run-data-schema-item]').should('have.length', 20);
 		})
-	})
+	});
+
+	it('can link and unlink run selectors between input and output', () => {
+		cy.fixture('Test_workflow_5.json').then((data) => {
```
Comment (OlegIvaniv) [inline cypress/e2e/5-ndv.cy.ts:169]: There's a command for this which also takes care about waiting for the WF to load `cy.createFixtureWorkflow('createFixtureWorkflow')`

**Review by OlegIvaniv — CHANGES_REQUESTED** [review]

File: cypress/e2e/5-ndv.cy.ts (hunk 2 above)
Comment (mutdmour) [reply cypress/e2e/5-ndv.cy.ts:169]: ah just copy paste.. will update

**Review by OlegIvaniv — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.226.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.226.1)

**Pull request merged**